sos/report/plugins/krb5.py

```diff
@@ -33,10 +36,32 @@ def setup(self):
             f"{self.kdcdir}/kdc.conf",
             "/var/log/kadmind.log"
         ])
+        self.collect_kinit()
         self.add_copy_spec("/var/log/krb5kdc.log", tags="kerberos_kdc_log")
         self.add_cmd_output(f"klist -ket {self.kdcdir}/.k5*")
         self.add_cmd_output("klist -ket /etc/krb5.keytab")
 
+    def collect_kinit(self):
+        """
+        Collect the kinit command output for the system with id_provider "AD"
+        or "IPA" domains.
+        """
+
+        hostname = socket.gethostname()[:11].upper()
+        hostname_fqdn = socket.getfqdn()
+        sssd_conf = "/etc/sssd/sssd.conf"
+        if os.path.isfile(sssd_conf):
+            with open(sssd_conf, 'r') as f:
+                for line in f:
+                    if re.match(r'\s*id_provider\s*=\s*ad',
+                                line, re.IGNORECASE):
+                        self.add_cmd_output(f"kinit -k '{hostname}$'")
+                        break
+                    elif re.match(r'\s*id_provider\s*=\s*ipa',
```

Comment:
as per the `pylint` message, replace the `elif` with just `if`

Reply:
Yes @arif-ali I missed that while doing the changes.